xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -365,7 +365,7 @@
 
 <br>
 <%  // Show Software Version if there is :
-    if (currentSess.getSoftwareVersion().size() > 1) {
+    if (currentSess.getSoftwareVersion().size() >=1) {
```

Comment:
Ah, good catch. Maybe use `!currentSess.getSoftwareVersion().isEmpty()`? That does the same, but is a bit clearer.